dump/src/reader/compat/v5_to_v6.rs

```diff
@@ -181,10 +181,8 @@ impl CompatV5ToV6 {
                     .indexes
                     .into_iter()
                     .map(|index| match index {
-                        v5::StarOr::Star => v6::StarOr::Star,
-                        v5::StarOr::Other(uid) => {
-                            v6::StarOr::Other(v6::IndexUid::new_unchecked(uid.as_str()))
-                        }
+                        v5::StarOr::Star => v6::IndexUidPattern::all(),
+                        v5::StarOr::Other(uid) => v6::IndexUidPattern::new_unchecked(uid.as_str()),
```

Comment:
Not sure it should be a `new_unchecked`.
If someone had an API key with something invalid, we should ignore his key and send a warning, I think?

Currently, let me know if I'm wrong, but `indexes_*_doggo` is a valid API key name that should be forbidden in the new version, right?

Reply:
Oh yes, you're right, perfect then :ok_hand: